src/textual/color.py

```diff
@@ -253,7 +258,9 @@ def parse(cls, color_text: str | Color) -> Color:
         """
         if isinstance(color_text, Color):
             return color_text
-        ansi_color = ANSI_COLOR_TO_RGB.get(color_text)
+        if color_text == "transparent":
+            return TRANSPARENT
```

Comment:
Kind of a psuedo-color. Might roll this in to the color names mapping at some point.